src/LinAlg/Test/MatrixTests.h

```diff
@@ -0,0 +1,227 @@
+//==============================================================================
+//!
+//! \file MatrixTests.h
+//!
+//! \date Apr 11 2016
+//!
+//! \author Eivind Fonn / SINTEF
+//!
+//! \brief Implementation of some unit tests for matrix.
+//!
+//==============================================================================
+
+#include <numeric>
+
+#include "MatVec.h"
+
+#include "gtest/gtest.h"
+
+#include <limits>
```

Comment:
Put this include after `<numeric>` grouping STL-headers

Reply:
heh, i put numeric after this instead. grouping in alpha order, with system headers after "our" headers to avoid problematic transitive include scenarios.